content/articles/restoring-domain.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Restoring a Domain
+excerpt: How to restore a domain with a DNSimple account.
+categories:
+- Domains
+---
+
+# Restoring a Domain
+
+To reactivate your domain that has entered the [redemption period](/articles/what-happens-when-domain-expires/#redemption-period), it has to be restored. After completion, your domain's registration will be extended by the minimum renewal period (usually 1 year).
+
+<warning>
+The registrar applies a mandatory restore fee.
+The final price of the restore will be the <strong>restore fee plus the standard renewal fee</strong>.
+</warning>
+
+##### To restore a domain
+
+1. Log in to DNSimple with your user credentials.
+1.  On the header, click the <label>Domains</label> tab, locate the relevant domain, and click on the name to access the domain page.
+1.  Select the <label>Restore domain</label> link from the registration status card.
+
+    ![Restore domain link](/files/restore-domain.png)
+
+1.  Confirm the price and click the <label>Restore Domain</label> button.
+
+    ![Restore domain form](/files/restore-domain-form.png)
+
+<info>
+The total restore price will be held on your card immediately.
+Your card will be charged once the restoration completes.
+If the restoration fails, funds will be released.
```

Comment:
We need to tell the customers that they will receive an email once the restore completes, or an email if it fails.